src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Header.scss'; 
+import hamburger from '../../Assets/svg/hamburger.svg'
+import logo from '../../Assets/png/logo.png'
+import magnifying_glass from '../../Assets/svg/magnifying_glass.svg'
+import icon from '../../Assets/svg/icon.svg'
+import shopping_bag from '../../Assets/svg/shopping_bag.svg'
+
+const Header = () => {
+  return (
+    <div>
+      <header className="header">
+
+        <img className='mobile_menu_button' src={hamburger} alt='mobile menu'/>
+
+        <img className='logo' src={logo} alt='logo'/>
+
+        <div className='links_container'>
+          <a className='big_desktop'>EPICURE</a>
+          <a className='sml_desktop'>Restaurants</a>
+          <a className='sml_desktop'>Chefs</a>
+        </div>
```

Comment:
- consider making the code more maintainable and scalable by using an array to store the links and mapping through them. This way, any future additions or changes to the header links can be easily managed in the array.

for example: 
```suggestion
<div className='links_container'>
{headerLinks.map((link) => (
    <a key={link.key} className={link.className}>
      {link.text}
    </a>
))}
</div>
```